src/assets/styles/index.scss

```diff
@@ -7,7 +7,8 @@
 @import "~styles/header";
 @import "~styles/event_list";
 @import "~styles/event_card";
-@import "~styles/pikaday";
+@import "~styles/vendor/pikaday";
+@import "~styles/vendor/mapbox-gl.css";
```

Comment:
this file is located at `node_modules/mapbox-gl/dist/mapbox-gl.css`. It didn't seem like a good idea to import stuff directly from the module itself, so I moved it here, but I'm also open to other things as this seems a little weird too.